models/shopify_holistic_reporting__customer_enhanced.sql

```diff
@@ -12,27 +12,28 @@ with shopify_customers as (
 
     select
         coalesce(shopify_customers.email, klaviyo_persons.email) as email,
-
-        coalesce(klaviyo_persons.full_name, shopify_customers.full_name) as full_name,
+        coalesce(klaviyo_persons.full_name, shopify_customers.first_name || ' ' || shopify_customers.last_name) as full_name,
         shopify_customers.customer_ids as shopify_customer_ids,
         klaviyo_persons.person_ids as klaviyo_person_ids,
         coalesce(shopify_customers.phone_numbers, klaviyo_persons.phone_numbers) as phone_number,
-        shopify_customers.first_shopify_account_made_at as shopify_customer_first_created_at,
-        shopify_customers.last_shopify_account_made_at as shopify_customer_last_created_at,
+        shopify_customers.first_account_created_at as shopify_customer_first_created_at,
+        shopify_customers.last_account_created_at as shopify_customer_last_created_at,
         klaviyo_persons.first_klaviyo_account_made_at as klaviyo_person_first_created_at,
         klaviyo_persons.last_klaviyo_account_made_at as klaviyo_person_last_created_at,
-        shopify_customers.last_updated_at as shopify_customer_last_updated_at,
+        shopify_customers.updated_timestamp as shopify_customer_last_updated_at,
         klaviyo_persons.last_updated_at as klaviyo_person_last_updated_at,
         shopify_customers.is_verified_email as is_shopify_email_verified,
+        shopify_customers.first_order_timestamp as shopify_first_order_at,
+        shopify_customers.most_recent_order_timestamp as shopify_last_order_at,
 
-        {{ dbt_utils.star(from=ref('int__shopify_customer_rollup'), relation_alias='shopify_customers', prefix='shopify_',
-                                except=['source_relation','email', 'full_name', 'customer_ids', 'phone_numbers', 'first_shopify_account_made_at','last_shopify_account_made_at', 
-                                        'last_updated_at', 'is_verified_email'] ) 
+        {{ dbt_utils.star(from=ref('shopify__customer_emails'), relation_alias='shopify_customers', prefix='shopify_', quote_identifiers=false,
+                                except=['source_relation', 'email', 'full_name', 'customer_ids', 'phone_numbers', 'first_account_created_at','last_account_created_at', 'last_updated_at',
```

Comment:
> looks like last_updated_at is removed from this CTE so is it needed?

so are you saying we should remove it from the `except` list? not sure exactly what ya mean 

> Also seems we're relying on first_name and last_name now from shopify_customers instead of full_name

so `full_name` is a field we used to create ourselves in the [int model](https://github.com/fivetran/dbt_shopify_holistic_reporting/blob/main/models/intermediate/int__shopify_customer_rollup.sql#L19) this PR deletes, so i just moved _where_ we perform the concatenation

Reply:
> so are you saying we should remove it from the except list? not sure exactly what ya mean

I think I'm looking at the renaming of `last_updated_at` to `updated_timetamp`, so maybe it doesn't have to be deleted but just renamed to `updated_timestamp` (If the intent is to bring in all the columns not already added into this model)?  Or does the `relation_alias` handle this by looking directly within the cte? 

> so full_name is a field we used to create ourselves in the [int model](https://github.com/fivetran/dbt_shopify_holistic_reporting/blob/main/models/intermediate/int__shopify_customer_rollup.sql#L19) this PR deletes, so i just moved where we perform the concatenation

Oh sorry I meant in reference to the except statement in the `dbt_utils.star` function, it seems like we'd bring in `first_name` and `last_name` but we've already produced `full_name`. However it looks like that was the earlier behavior so it's probably fine the way it is.

Reply:
ohhhhhh yes i will add `updated_timetamp` to the except list! i should also add `created_timestamp`. the timestamp fields got renamed in a recent version of shopify, so they got all jumbled up together

i am actually going to revert line 23 though just for consistency (`last_updated_at` is also just calculated in a more reader-friendly way than `updated_timestamp`) 

> Oh sorry I meant in reference to the except statement in the dbt_utils.star function, it seems like we'd bring in first_name and last_name but we've already produced full_name. However it looks like that was the earlier behavior so it's probably fine the way it is.

Ahh sorry I see what you mean!  i'll replace `full_name` in the except list with `first_name, last_name`

Reply:
Sounds great! Thanks for the walkthrough. All the new changes look good!